Bottom: 3a913e2d815b0b9c94166832305c2dd5f1c5d6cb
Top:    549d79f13bafe1e21e24c41644adddbf02e76cf0
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 18:30:24 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index c06f240..2864898 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2162,7 +2162,7 @@ int main(int argc, char *argv[])
 
 	if (dumpmaps) {
 		struct master_mapent *entry;
-		struct list_head *p;
+		struct list_head *head, *p;
 		struct mapent_cache *nc;
 
 		open_log();
@@ -2180,14 +2180,15 @@ int main(int argc, char *argv[])
 		lookup_nss_read_master(master_list, 0);
 		master_show_mounts(master_list, dump);
 
-		p = master_list->next;
-		while (p != master_list) {
-			entry = list_entry(p, struct master_mapent, join);
+		/*head = &master_list->mounts;
+		p = head->next;
+		while (p != head) {
+			entry = list_entry(p, struct master_mapent, list);
 			p = p->next;
 			master_free_mapent_sources(entry, 1);
 			master_free_mapent(entry);
 		}
-		master_kill(master_list);
+		master_kill(master_list);*/
 		if (dump)
 			free((void *) dump);
 		exit(0);
